src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int NAME_INDEX = 0;
+    private static final int PARAM_INDEX = 1;
+    private static final int RESULT_INDEX = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeWtoFile(createReport(readFromFile(fromFileName)), toFileName);
+    }
+
+    public List<String> readFromFile(String fromFileName) {
+        List<String> fileLines;
+        try {
+            fileLines = Files.readAllLines(Paths.get(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
+        }
+        return fileLines;
```

Comment:
```suggestion
        try {
            return  Files.readAllLines(Paths.get(fromFileName));
        } catch (IOException e) {
            throw new RuntimeException("Can't read data from file: " + fromFileName, e);
        }
```

Reply:
If I do not return anything from the method, then I will have nothing to pass to the next method. Either I misunderstood the point of the remark, or the remark does not make sense.